sdk/core/core-amqp/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should we extend the base one? 

```
  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
```

Reply:
Hey Jeremy! I noticed that this package didn't have a ```.eslintrc.json``` file, so I made my own with the rule. I'm not exactly sure what else to put - is there anything else I'm missing?

Reply:
I believe this is the right place but the content should be similar to the one that you modified in the other PR #19966 https://github.com/Azure/azure-sdk-for-js/blob/edeb77f0d67c058af7571ccf85174b2704da1e19/sdk/core/core-tracing/.eslintrc.json

So you can just copy the "plugins" and "extends" fields here.

Reply:
Will do!